### PR TITLE
Added displayName prop to AsseTree

### PR DESCRIPTION
**Pull request by ArbenPaloja**

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-444.gearboxjs.preview.cogniteapp.com

**Comment by polomani** [comment]
please, run `yarn lint --fix` and `yarn test` first
the build is failing

**Comment by f1cognite** [comment]
@polomani Fixed in latest commit

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-444.gearboxjs.preview.cogniteapp.com

File: src/components/AssetTree/stories/full.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ No required props
 
 | Property              | Description                                 | Type                        | Default |
 | --------------------- | ------------------------------------------- | --------------------------- | ------- |
+| `displayName`         | Modify the display name of an asset         | `(asset: Asset) => string` | `Name: Description` |
```
Comment (polomani) [inline src/components/AssetTree/stories/full.md]:
```suggestion
| `displayName`         | Modify the display name of an asset         | `(asset: Asset) => string` | `(...) => "assetName: assetDescription" |
```

File: src/components/AssetTree/AssetTree.tsx [hunk 2]
```diff
@@ -76,9 +79,23 @@ class AssetTree extends React.Component<AssetTreeProps, AssetTreeState> {
...
-  static returnPretty(asset: Asset) {
+  static getDisplayName(
+    asset: Asset,
+    displayNameFunc?: (asset: Asset) => string
+  ) {
+    if (displayNameFunc) {
+      return displayNameFunc(asset);
+    }
+    return `${asset.name}${asset.description ? ':' : ''} ${asset.description ||
+      ''}`;
```
Comment (polomani) [inline src/components/AssetTree/AssetTree.tsx]: return `${asset.name}${asset.description ? ':' + asset.description : ''}`

**Comment by polomani** [comment]
Should we add a test for it (I know the coverage has been increased, but that's just by accident)

File: src/components/AssetTree/stories/displayName.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+import 'antd/dist/antd.css';
+
+import React from 'react';
+import { AssetTree, OnSelectAssetTreeParams } from '@cognite/gearbox';
+
+function ExampleComponent(props) {
+
+  return (
+    <AssetTree
+      displayName={(asset) => \`Id: ${asset.id}\`}
```
Comment (polomani) [inline src/components/AssetTree/stories/displayName.md]:
```suggestion
      displayName={asset => \`Id: ${asset.id}\`}
```

File: src/components/AssetTree/stories/AssetTree.stories.tsx [hunk 4]
```diff
@@ -114,6 +115,17 @@ storiesOf('AssetTree/Examples', module)
       },
     }
   )
+  .add(
+    'Custom display name',
+    () => {
+      return <AssetTree displayName={asset => `Id: ${asset.id}`} />;
```
Comment (polomani) [inline src/components/AssetTree/stories/AssetTree.stories.tsx]:
```suggestion
      return <AssetTree displayName={({id})=> `Id: ${id}`} />;
```

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-444.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-444.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-444.gearboxjs.preview.cogniteapp.com

File: package.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cognite/gearbox",
-  "version": "1.2.0",
+  "version": "1.3.0",
```
Comment (polomani) [inline package.json]:
```suggestion
  "version": "1.2.0",
```

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-444.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-444.gearboxjs.preview.cogniteapp.com

**Pull request merged**